pangeo_forge_orchestrator/cli.py

```diff
@@ -1,65 +1,86 @@
-import ast
-import os
+import json
+from dataclasses import dataclass
 
 import typer
 
 from .client import Client
 
-cli = typer.Typer()
+cli = typer.Typer(name="Pangeo Forge")
 db = typer.Typer()
 
 top_endpoint = "A top level endpoint, enclosed in forward slashes, e.g. '/my_endpoint/'."
 specific_endpoint = "A unique entry endpoint concluding with an integer id, e.g. '/my_endpoint/1'."
 json_help = "A JSON string, created via with Python's ``json.dumps`` method, for example."
 
 
-def get_client():
+def get_client(server, api_key):
     # Putting this in a function means that the environment variable is checked
     # at evaluation time rather than import time.
-    return Client(base_url=os.environ["PANGEO_FORGE_DATABASE_URL"])
+    return Client(base_url=server, api_key=(api_key or None))
+
+
+@dataclass
+class Common:
+    server: str
+    api_key: str
```

Comment:
It looks like this class is never called. Is it used for something or perhaps it was a relic of a since-abandoned implementation idea?

Reply:
Good catch, will remove.